src/etc/inc/openvpn.auth-user.php

```diff
@@ -186,15 +186,13 @@ function getNasIP() {
 }
 
 if (isset($attributes['framed_ip'])) {
-/* XXX: only use when TAP windows driver >= 8.2.x */
-/*	if (isset($attributes['framed_mask'])) {
+	if (isset($attributes['framed_mask'])) {
 		$content .= "topology subnet\n";
 		$content .= "ifconfig-push {$attributes['framed_ip']} {$attributes['framed_mask']}";
 	} else {
-*/
-	$content .= "topology net30\n";
-	$content .= "ifconfig-push {$attributes['framed_ip']} ". long2ip((ip2long($attributes['framed_ip']) + 1));
-//	}
+		$content .= "topology net30\n";
+		$content .= "ifconfig-push {$attributes['framed_ip']} ". long2ip((ip2long($attributes['framed_ip']) + 1));
```

Comment:
As long as this is being fixed, we may as well fix the consistency in client/server address as well. In client-specific config/overrides the server address is made one less than the client address, not one higher, which is consistent with OpenVPN's own internal address assignment.  See https://github.com/pfsense/pfsense/blob/master/src/etc/inc/openvpn.inc#L1053 - thoughts?

Should be as easy and changing that +1 to -1.